database/migrations/2023_10_30_034209_create_activity_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('activity', function (Blueprint $table) {
+            $table->id('activity_id');
+            $table->foreignId('activity_type_id')->constrained(
+                table: 'activity_type', column: 'activity_type_id'
+            );
+            $table->year('academic_year');
```

Comment:
I think academic_year should just be a string instead of year.

(eg. 'A.Y. 2020-2021' or just '2020-2021')